source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +224,53 @@ export default class WindowManager {
       return
     }
 
+    const platformIcons: {[prop: string] : string} = {
+      'darwin': '/png/16x16.png',
```

Comment:
On MacOS tray icon should be 22x22 with a white logo and transparent background as per the [tray design](https://github.com/UofA-SPI21-team7/zettlr-assignment/wiki/Tray-Design). I suggest put:

```typescript
      'darwin': '/png/22x22_white.png',
```

Also add the file `22x22_white.png` (see slack channel, or [this link to file](https://univ-adl-sw-proj.slack.com/files/U01S1UV729W/F020HE3FL4F/22x22_white.png)) to the `Zettlr\resources\icons\png` folder.